core/src/main/scripts/importer/validateData.py

```diff
@@ -1774,9 +1783,10 @@ def checkHeader(self, cols):
 
         for col_index, col_name in enumerate(self.cols):
             if not col_name.isupper():
-                self.logger.warning(
-                    "Clinical attribute name not in all caps",
+                self.logger.error(
```

Comment:
when giving an error, please specify in the error message, or in the code documentation just above the error, what would otherwise break in cBioPortal.

Reply:
Done